platform/mds/mds/src/main/java/org/motechproject/mds/ex/object/DuplicateEntryException.java

```diff
@@ -0,0 +1,31 @@
+package org.motechproject.mds.ex.object;
+
+import org.motechproject.mds.ex.MdsException;
+
+import javax.jdo.JDODataStoreException;
+import java.sql.SQLIntegrityConstraintViolationException;
+
+/**
+ * Signals that it was not possible to update object instance from the provided data.
+ */
+public class DuplicateEntryException extends MdsException {
```

Comment:
I've noticed we have some constraint recognition logic in ObjectCreateException and ObjectUpdateException, can we rely on those?